### PR TITLE
remove raw-loader as a dependency

### LINKED ISSUE #4997: Raw-loader needs to be replaced with asset modules in webpack 5
This seems to have broken master and needs to be reverted/fixed by using asset modules with webpack 5.

_Originally posted by @shefalijoshi in https://github.com/nasa/openmct/issues/4978#issuecomment-1078441116_

More here
https://webpack.js.org/guides/asset-modules/

### PR DESCRIPTION
**Pull request by scottbell**
Closes #4997 

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Changes appear not to be breaking changes?
* [ ] Appropriate unit tests included?
* [x] Code style and in-line documentation are appropriate?
* [x] Commit messages meet standards?
* [x] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [x] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: e2e/tests/example/generator/SinewaveLimitProvider.e2e.spec.js [hunk 1]
```diff
@@ -157,5 +157,9 @@ test.describe('Sine Wave Generator', () => {
                 y: 28
             }
         });
+
+        // Verify that where we click on canvas shows the number we clicked on
+        await expect(page.locator('.value-to-display-nearestValue')).toContainText('0.00');
```
Comment (scottbell) [inline e2e/tests/example/generator/SinewaveLimitProvider.e2e.spec.js:162]: clicking on the canvas, and then seeing if we get a nearest value in it means we have a real, rendered plot.

File: example/generator/WorkerInterface.js [hunk 2]
```diff
@@ -21,20 +21,13 @@
...
-    var workerBlob = new Blob(
-        [workerText],
-        {type: 'application/javascript'}
-    );
-    var workerUrl = URL.createObjectURL(workerBlob);
-
-    function WorkerInterface() {
+    function WorkerInterface(openmct) {
+        // eslint-disable-next-line no-undef
+        const workerUrl = `${openmct.getAssetPath()}${__OPENMCT_ROOT_RELATIVE__}generatorWorker.js`;
```
Comment (scottbell) [inline example/generator/WorkerInterface.js:30]: this is how [CouchObjectProvider](https://github.com/nasa/openmct/blob/master/src/plugins/persistence/couch/CouchObjectProvider.js#L53) and [InMemorySearchProvider]https://github.com/nasa/openmct/blob/master/src/api/objects/InMemorySearchProvider.js#L170 do it.

File: karma.conf.js [hunk 3]
```diff
@@ -38,6 +38,10 @@ module.exports = (config) => {
             {
                 pattern: 'dist/inMemorySearchWorker.js*',
                 included: false
+            },
+            {
+                pattern: 'dist/generatorWorker.js*',
+                included: false
```
Comment (scottbell) [inline karma.conf.js:44]: needed for tests to make the worker available at test time

File: webpack.common.js [hunk 4]
```diff
@@ -18,6 +18,7 @@ const gitBranch = require('child_process')
 const config = {
     entry: {
         openmct: './openmct.js',
+        generatorWorker: './example/generator/generatorWorker.js',
```
Comment (scottbell) [inline webpack.common.js:21]: don't webpack the worker so we can load it from an URL

File: package.json [hunk 5]
```diff
@@ -57,7 +57,6 @@
     "plotly.js-basic-dist": "2.5.0",
     "plotly.js-gl2d-dist": "2.5.0",
     "printj": "1.3.1",
-    "raw-loader": "4.0.2",
```
Comment (unlikelyzero) [inline package.json:60]: nice!

File: webpack.common.js (hunk 4 above)
Comment (unlikelyzero) [reply webpack.common.js:21]: note: i believe this change might be necessary on our closed source webpack configs. VIPER and maybe VISTA @davetsay ?

Comment (shefalijoshi) [reply webpack.common.js:21]: Only if they use the generatorWorker

**Pull request merged**